impl/otbn/montmul.i.vad

```diff
@@ -134,6 +137,30 @@ procedure montmul(ghost vars: mvars)
         mvars_inv(vars, heap, NA, x19, x16, x17, NA, NA);
         montmul_inv(a_view, vars.x_it.buff, NUM_WORDS, vars.y_it.buff, vars.rsa);
 {
+    /* TODO: Can this header comment appear above the procedure? Better yet, can
```

Comment:
Interesting. With some tweaks the comment can probably be added to the procedure's constructor. Dumping pre/post conditions might involve some modifications to Vale.